src/main/java/uk/gov/companieshouse/officerfiling/api/exception/CompanyAppoinmentServiceException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.officerfiling.api.exception;
+
+/**
+ * Appointment not found or external query failed.
+ */
+public class CompanyAppoinmentServiceException extends RuntimeException{
```

Comment:
We've mis-spelled this - CompanyAppoin **t** mentServiceException
Could you do a quick push to fix the spelling here and where it's used?